documentation/changelog.rst

```diff
@@ -11,11 +11,11 @@ New features
 -------------
 * Add basic sensor info to sensor page [see `PR #1115 <https://github.com/FlexMeasures/flexmeasures/pull/1115>`_]
 * Support zoom-in action on the asset and sensor charts [see `PR #1130 <https://github.com/FlexMeasures/flexmeasures/pull/1130>`_]
- 
+* Allow three ways of passing a variable quantity in most of the ``flex-model`` and ``flex-context`` fields [see `PR #1127 <https://github.com/FlexMeasures/flexmeasures/pull/1127>`_]
```

Comment:
I'm fine with mentioning the class, but the responsible PR is already mentioned explicitly.

Reply:
I understand your comment better now. You're describing the potential need to find the PR when you have the class in front of you and are looking for the PR that introduced it, but are searching in the changelog rather than using `git blame`.